tests/testthat/test_manifold.R

```diff
@@ -1 +1,26 @@
 # Unit tests for Manifold.
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+  rand <<- ceiling(10*runif(1))
+  manifold <<- Manifold$new(dimension = rand)
+
```

Comment:
Too many blank lines. I think 2 lines is the standard before the definition of a function?